app/client/src/utils/mapFunctions.js

```diff
@@ -15,7 +15,7 @@ import { getSelectedCommunityTab } from 'utils/utils';
 
 const waterbodyStatuses = {
   good: { condition: 'good', label: 'Good' },
-  polluted: { condition: 'polluted', label: 'Impaired' },
+  polluted: { condition: 'polluted', label: 'Impaired\n(Issues Identified)' },
```

Comment:
```suggestion
  polluted: { condition: 'polluted', label: 'Impaired (Issues Identified)' },
```

The `\n` can be removed, since it seems like the browser is stripping it of anyways.

Reply:
I can do that. It's actually in there for the WaterbodyInfo pop-ups, where I included the rule:
```
th, td {
  white-space: pre-wrap;
}
```
in line 86 of `WaterbodyInfo.js`, so that's the only place it's visible.

Reply:
Ahh ok. That can stay.

Reply:
Sorry to flip on this. I think the `\n` should be removed.

The styles associated with this have affected a bunch of other stuff (i.e., monitoring station content, dischargers, etc.).